roles/openshift_health_checker/library/etcdstats.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+'''
+Ansible module that determines if the size of paths in an etcd cluster exceed a given limit.
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+IMPORT_EXCEPTION = None
+
+try:
+    import etcd
+# pylint: disable=invalid-name
+except ImportError as IMPORT_EXCEPTION:
+    # Allows for test mocking
```

Comment:
The problem is not the comment, but the assignment of etcd to a fake in this block. We should only construct fakes in the testing code. If production code is calling this module and the import fails, the module execution should fail with an appropriate error message.